contentful/content-management/import-content.js

```diff
@@ -0,0 +1,30 @@
+require('dotenv').config();
+
+const contentfulImport = require('contentful-import');
+const fs = require('fs');
+
+async function importContentfulData() {
+  const options = {
+    contentFile: process.env.CONTENT_FILE,
+    spaceId: process.env.SPACE_ID,
+    managementToken: process.env.MANAGEMENT_TOKEN,
+
+    // the default env is master
+    environmentId: process.env.ENVIRONMENT,
```

Comment:
I think it'd be worthwhile for us to have a helper method somewhere in the content-management folder, `getEnvironment` or similar that validates ENVIRONMENT against a list of the options for it and errors if its not one of the options so it doesn't default this way

Reply:
Ah yeah, something more like this? [aeac9aa](https://github.com/DFE-Digital/sts-plan-technology-for-your-school/pull/871/commits/aeac9aa03d6beafeadab7ca5f0bf0d7f20d8c2cb)

Reply:
Yeah that'll work, thanks